packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -1746,20 +1791,45 @@ class _DefaultSemanticsGestureDelegate extends SemanticsGestureDelegate {
         horizontal == null
             ? null
             : (DragUpdateDetails details) {
-              horizontal.onDown?.call(DragDownDetails());
-              horizontal.onStart?.call(DragStartDetails());
+              final Offset localCenter = _getLocalRectFromRenderObject(renderObject).center;
+              final Offset globalCenter = _getGlobalRectFromRenderObject(renderObject).center;
+              final Offset newLocalOffset = localCenter + details.delta;
+              final Offset newGlobalOffset = globalCenter + details.delta;
```

Comment:
Would it be more robust to transform newLocalOffset to global coordinates instead of adding directly to globalCenter?

Reply:
Could be a bug if the renderObject is scaled or rotated at least.

Reply:
yes  you are right, looks like the delta is in local coordinate